.github/workflows/test_and_release.yml

```diff
@@ -39,6 +39,9 @@ jobs:
       - name: Run jshint
         run: npm run jshint
 
+      - name: Run fmt
```

Comment:
We probably shouldn't format here but make sure that formatting already is correct.

Reply:
This will cause it to fail if it's not correct. This enforces that the style is followed.

If you don't run this, you can't enforce that it was done before (unless you do something else that's effectively the same).

Reply:
So it doesn't format itself but only checking that the format is correct?

Reply:
That's correct. It will `exit 1` if there are any changes and `exit 0` if there are no changes.